Modules/posixmodule.c

```diff
@@ -8202,8 +8202,10 @@ os_sched_get_priority_max_impl(PyObject *module, int policy)
 {
     int max;
 
+    /* make sure that errno is cleared before the call */
+    errno = 0;
     max = sched_get_priority_max(policy);
```

Comment:
In C99 you can declare variable where you need it.
```suggestion
    /* make sure that errno is cleared before the call */
    errno = 0;
    int max = sched_get_priority_max(policy);
```